apps/emqx_dashboard/test/emqx_dashboard_monitor_SUITE.erl

```diff
@@ -660,19 +655,142 @@ t_persistent_session_stats(Config) ->
 
     ok.
 
+%% Checks that we get consistent data when changing the requested time window for
+%% `/monitor'.
+t_smoke_test_monitor_multiple_windows(Config) ->
```

Comment:
This tests seems to capture well the issue without the patch: if you check the connection count on each `?retry`, it continually increases and reaches ~ 28 connections before failing.